src/ansys.jl

```diff
@@ -41,6 +41,7 @@ matrix elements. We expect these to be strings `parse(Float64, _)` can handle.
 =#
 
 using LinearAlgebra: diagind
+using Compat
```

Comment:
Curious why this was needed

Reply:
Turns out `isnothing` wasn’t implemented yet in julia 1.0